doc/10-icinga-template-library.md

```diff
@@ -2259,6 +2259,42 @@ snmp_service_count         | **Optional.** Compare matching services with a spec
 snmp_service_showall       | **Optional.** Show all services in the output, instead of only the non-active ones. Defaults to false.
 snmp_service_noregexp      | **Optional.** Do not use regexp to match NAME in service description. Defaults to false.
 
+### Elasticsearch Netways <a id="plugin-check-command-elasticsearch-netways"></a>
+
+This category includes all subcommands using the elasticsearch-netways plugin.
+
+Custom variables passed as [command parameters](03-monitoring-basics.md#command-passing-parameters):
+
+Name                    	          | Description
+------------------------------------------|-----------------------------------------------------------------------------------------
+elasticsearch_hostname     | **Optional.** Hostname of the Elasticsearch instance (default 'localhost').
+elasticsearch_port         | **Optional.** Port of the Elasticsearch instance (default 9200).
+elasticsearch_username     | **Optional.** Username if authentication is required. No value defined as default.
+elasticsearch_password     | **Optional.** Password if authentication is required. No value defined as default.
+elasticsearch_tls          | **Optional.** Use a HTTPS connection (default false).
+elasticsearch_insecure     | **Optional.** Skip the verification of the server's TLS certificate (default false).
+elasticsearch_timeout      | **Optional.** Timeout in seconds for the CheckPlugin (default 30).
+
+#### Elasticsearch Query <a id="plugin-check-command-elasticsearch-netways-query"></a>
+
+This subcommand is provided by [elasticsearch netways](https://github.com/NETWAYS/check_elasticsearch#query)
+and executed as `elasticsearch query`. Checks the total hits/results of an Elasticsearch query.
+
+Custom variables passed as [command parameters](03-monitoring-basics.md#command-passing-parameters):
+
+Name                    	          | Description
+------------------------------------------|-----------------------------------------------------------------------------------------
+elasticsearch_query          | **Required.** The Elasticsearch query.
+elasticsearch_query_index    | **Optional.** Name of the Index which will be used (default '_all').
+elasticsearch_query_msgkey   | **Optional.** Message of messagekey to display.
+elasticsearch_query_msglen   | **Optional.** Number of characters to display in the latest message (default 80).
+elasticsearch_query_warning  | **Optional.** Warning threshold for total hits (default '20').
+elasticsearch_query_critical | **Optional.** Critical threshold for total hits (default '50').
```

Comment:
You quote some numbers while not quoting others.

Reply:
The threshold values for critical and warning are string values

Reply:
All CLI args to all programs are strings. Please use the usual notation of the rest of this file.

Reply:
Done

Reply:
I don't think so.

Reply:
And in the check command itself?

Reply:
The check plugin handles the threshold like the [Nagios plugin guidelines](https://nagios-plugins.org/doc/guidelines.html#THRESHOLDFORMAT). So it has to be string. If this format is no valid for you, then I'll close this PR.